plugin/pkg/controllerinstallation/admission.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
I also vote to do not introduce this admission plugin. The change in the controllerregistration control to do not create new controllerinstallations when the ctrlreg has a `deletionTimestamp` should be enough.

Reply:
Removed